charts/seed-bootstrap/charts/dependency-watchdog/templates/probe-networkpolicy.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: {{ include "networkpolicyversion" . }}
```

Comment:
This seems a very generic network policy. Can we move it somewhere else and change the description (remove dependency watchdog)?

Reply:
It is generic but no other component requires this particular pattern of egress (so far).

I thought about moving it to [`charts/global-network-policies`](https://github.com/gardener/gardener/tree/master/charts/global-network-policies). But wouldn't that get applied to all other (shoot control-plane) namespaces too where they are currently not needed?

Reply:
They might not be needed today, sure, but this network policy looks quite general to me. Hence, moving it to a general location now makes it easier to use it later on if required by another component.